pkg/sql/parser/table_name.go

```diff
@@ -261,7 +261,7 @@ type TableNameWithIndex struct {
 
 // Format implements the NodeFormatter interface.
 func (n TableNameWithIndex) Format(buf *bytes.Buffer, f FmtFlags) {
```

Comment:
Should this (and anything else that contains a `NormalizableTableName`) also be a reference? I'm not sure when this can come up but could we also want this to persist its normalization?